src/components/formComponents/Form.js

```diff
@@ -0,0 +1,42 @@
+import { Center, Button, Box } from "@chakra-ui/react";
+import { useState } from "react";
+import { Link, useParams } from "react-router-dom";
+
+const Form = ({ formData }) => {
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    // TODO - send data somewhere
+  }
```

Comment:
Andá pensando cómo vas a hacer para en este punto poder recuperar el estado de todos los Inputs del Form y armar una pegada para el backend. Eso mínimo debería estar parametrizado desde los child form y ser transparente para este componente.